gard_owl_ingest/mondo_mapping_status.py

```diff
@@ -242,21 +242,33 @@ def gard_mondo_mapping_status(
     outpath_mondo_sssom_curate = RELEASE_DIR / f'gard-mondo{file_suffix}_curation.sssom.tsv'
     write_tsv_with_comments(sssom_curate_df, MONDO_SSSOM_METADATA_PATH, outpath_mondo_sssom_curate)
 
-    # - gard-mondo.robot.template.sssom.tsv
+    # - mondo-gard.robot.template.sssom.tsv
```

Comment:
Completed! Here's a copy/paste of [my comment in the issue](https://github.com/monarch-initiative/gard/issues/15#issuecomment-1672327656).

---

You can find the template in the new release [here](https://github.com/monarch-initiative/gard/releases/tag/v2023-08-09). Here's a snippet:

|mondo_id     |type     |gard_id             |source_external   |subset                                        |
|-------------|---------|--------------------|------------------|----------------------------------------------|
|ID           |TYPE     |A oboInOwl:hasDbXref|>A oboInOwl:source|AI oboInOwl:inSubset                          |
|MONDO:0000023|owl:Class|GARD:17820          |Orphanet:464724   |http://purl.obolibrary.org/obo/mondo#gard_rare|
|MONDO:0000208|owl:Class|GARD:18459          |OMIM:616033       |http://purl.obolibrary.org/obo/mondo#gard_rare|
|MONDO:0000209|owl:Class|GARD:17882          |Orphanet:486811   |http://purl.obolibrary.org/obo/mondo#gard_rare|

Reply:
## About the `-exact` artefacts (including `mondo-gard-exact.robot.template.sssom.tsv`)
You'll notice lots of other files in the release.

I ran `make all`, which auto-deployed it afterwards (not sure if that should be automatically done within the `all` goal but that's what I had before).

You'll also notice that there is a second, `-exact` template: `mondo-gard-exact.robot.template.sssom.tsv`. This is because I run the pipeline twice, once where I only included `skos:exactMatch`, and one where I don't filter out any predicates. So, some artefacts have a second one for `-exact`.

However, there are no differences between `mondo-gard-exact.robot.template.sssom.tsv` and `mondo-gard.robot.template.sssom.tsv`, as the template is created using `gard-mondo.sssom.tsv` as a source, and that one has no differences with `gard-mondo-exact.sssom.tsv`. I don't remember why they have no differences though, but I believe it is expected behavior.

Reply:
I dont think we nee this `-exact` in the file name - this just raises questions what the other filename is supposed to be (without the exact).

Reply:
That makes sense, but the question is what to do:
a. keep both sets of artefacts, `-exact`, and append the affix `-all-predicates` to the other set
b. Drop the `all-predicates` ones, and just keep the `-exact` ones, and just remove the `-exact` affix.

I'm guessing you might want 'b' but I'm fine with either. Both of them are easy to implement.

Reply:
For the template, we only care about "exact" - you can drop the other file. For the sssom file (if any), you can add the word "exact" to the exact one, and leave the other one without a qualifier! Thanks for checking.

Reply:
Done! https://github.com/monarch-initiative/gard/releases/tag/v2023-08-13